.travis.yml

```diff
@@ -5,32 +5,18 @@ git:
 
 jobs:
   include:
-    - name: Vim 7.4
-      env:
-      - VIM_VERSION=v7.4
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-v7.4.txt
-    - name: Vim v8
-      env:
-      - VIM_VERSION=v8.0.0000
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-v8.0.txt
-    - name: Vim master
-      env:
-      - VIM_VERSION=master
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-master.txt
```

Comment:
why did you remove these tests?

Reply:
Will be reverted - only to have a single (relevant) job here.

Reply:
okay 👍